fvm/evm/types/result.go

```diff
@@ -132,8 +132,7 @@ func (res *Result) Receipt() *gethTypes.Receipt {
 		return nil
 	}
 	receipt := &gethTypes.Receipt{
-		Type:              res.TxType,
```

Comment:
Not sure how it was working before but this was problematic to construct receipt roots for blocks that included a direct call type.